src/extension.ts

```diff
@@ -14,6 +14,9 @@ export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(vscode.languages.registerHoverProvider({ scheme: "file" }, { provideHover }));
 }
 
+// A Map to store cached hover content
+const hoverContentCache: Map<string, vscode.Hover> = new Map();
```

Comment:
You can remove the comments


```suggestion

const hoverContentCache: Map<string, vscode.Hover> = new Map();
```